perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -1962,6 +1962,28 @@ public String call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Send reminder of username in the given namespace to user's preferred email address.
+	 *
+	 * @param member int Member to get user to send link mail to
+	 * @param namespace String Namespace to change password in (member must have login in it)
+	 * @param emailAttributeURN String URN of the attribute with stored mail
+	 * @param language String 2-char language code of the message
+	 */
+	sendUsernameReminder {
```

Comment:
Both `sendAccountActivationLinkEmail` and `sendPasswordResetLinkEmail` are extracted to openapi and I feel like this method might also eventually be called via the new GUI app. Could you please add it to openapi?

Reply:
Added, thanks for the suggestion.